packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -119,7 +119,7 @@ export const PersonalDetailsForm = observer(({ history }: { history: BrowserHist
         setStatus({ msg: '' });
         const request = makeSettingsRequest({ ...values }, residence_list, states_list, is_virtual);
         setIsBtnLoading(true);
-        const data = await WS.setSettings(request);
+        const data = await WS.authorized.storage.setSettings(request);
```

Comment:
why do we need .storage here.isnt it used for caching?

Reply:
@amina-deriv , I'll double check it

Reply:
thanks for noticing it @amina-deriv ,
removing it